test/extended/router/headers.go

```diff
@@ -113,6 +113,7 @@ func dumpRouterHeadersLogs(oc *exutil.CLI, name string) {
 func getRoutePayloadExec(ns, execPodName, url, host string) (string, error) {
 	cmd := fmt.Sprintf(`
 		set -e
+		rc=0
```

Comment:
I was going to say that initializing `rc` is unnecessary because there is no loop, but it's probably a good idea anyway in case the parent process set `rc`.